src/lib/selectors.ts

```diff
@@ -382,4 +384,17 @@ const selectors: Record<number, { readonly selector: bigint; readonly name?: str
   },
   // end:generate
 }
+
+export const aptosSelectors: Selectors = {
+  '1': { name: 'aptos-mainnet', selector: 4741433654826277614n },
+  '2': { name: 'aptos-testnet', selector: 743186221051783445n },
+  '4': { name: 'aptos-localnet', selector: 4457093679053095497n },
+}
+
+const selectors: Selectors = { ...evmSelectors, ...aptosSelectors }
```

Comment:
Hmm, Aptos chainIds will conflict with EVM's, and override them here.

We will also need to adjust all functions accepting a `chainId: number` (e.g. [chainSelectorFromId](https://github.com/smartcontractkit/ccip-tools-ts/blob/3af48d6774fe040c8b64d0038e84ccb7f43734d4/src/lib/utils.ts#L251-L253), to be able to handle conflicting chainIds on different families.

I think we could add a `ChainFamily` enum, and make the selectors `Record` have a `${ChainFamily}:${number}` key (`string`), e.g. `evm:1` for `ethereum-mainnet`.

But I really wanted to minimize architecture changes in this PR; it was mostly intended to add just an Aptos hasher, and is growing in scope and changes.

Reply:
> Hmm, Aptos chainIds will conflict with EVM's, and override them here.
Right! Let me fix that

To avoid more major changes in this PR we can just export the `isAptosChain` function, as is the only one used, and keep `selectors` as evm only for now.

We can add support for different chain families later on